Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/RemoteXDAInstance.tsx

```diff
@@ -103,16 +103,10 @@ function RemoteXDAInstance(props: IProps) {
             <hr />
```

Comment:
Seems to no longer draw a line. Something about changing the parent styling likely lead to this. Looking at the element, it looks like it got it's width overwritten. Setting the width to 100% in the styling seems to fix it.